tools/cli/docs/cli/wallet.md

```
@@ -0,0 +1,369 @@
+```bash
+# Create a testnet wallet
+galaxy wallet create my-wallet
+
+# Create a mainnet wallet
+galaxy wallet create production-wallet --network mainnet
+
+# Create and output JSON
+galaxy wallet create api-wallet --json
+```
+
+**Output:**
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)